doc/develop/sca/codechecker.rst

```diff
@@ -42,6 +42,23 @@ To configure CodeChecker or analyzers used, arguments can be passed using the
     -DCODECHECKER_ANALYZE_OPTS="--config;$CODECHECKER_CONFIG_FILE;--timeout;60"
 
 
+Storing CodeChecker results
+***************************
+
+If a CodeChecker server is active, the results can be uploaded using the optional
+``CODECHECKER_STORE`` or ``CODECHECKER_STORE_OPTS`` parameters, e.g.
+
+.. code-block:: shell
+
+    west build -b mimxrt1064_evk -s samples/basic/blinky -- -DZEPHYR_SCA_VARIANT=codechecker \
```

Comment:
The `-s` is not needed, as specifying source dir that way is not really needed anymore. 
In fact, could you sneak in a commit also removing it from the couple other places in this file?